scripts/qa_crowbarsetup.sh

```diff
@@ -2304,6 +2330,15 @@ function oncontroller_testsetup()
     if ! ssh $vmip curl www3.zq1.de/test ; then
         complain 95 could not reach internet
     fi
+
+    # skip volume tests for docker
+    if [ "$image_name" == "cirros" ]; then
+        echo "RadosGW Tests: $radosgwret"
+        echo "Tempest: $tempestret"
+        echo "Volume in VM: skipped (not supported for docker)"
+        test $tempestret = 0 -a $radosgwret = 0 || exit 102
```

Comment:
I'm a bit worried that we're not removing the floating IP here. Maybe we should instead do something like:

```
if [ -z "$want_docker" ] ; then
  # do volume tests
fi

# cleanup so that we can run testvm without leaking volumes, IPs etc
...
```